doc/js_tutorials/js_setup/js_setup/js_setup.markdown

```diff
@@ -32,6 +32,8 @@ source ./emsdk_env.sh
 echo ${EMSCRIPTEN}
 @endcode
 
+The version 1.39.16 of emscripten is verified for latest WebAssembly. Please check the version of emscripten to use the newest features of WebAssembly.
+
```

Comment:
I think the install commands should be modified to
```
./emsdk update
./emsdk install 1.39.16
./emsdk activate 1.39.16
```

Reply:
Done